giraffe/src/types/index.ts

```diff
@@ -163,6 +167,14 @@ export interface CustomLayerRenderProps {
   columnFormatter: (colKey: string) => (x: any) => string
 }
 
+export interface RawFluxDataTableLayerConfig {
+  type: 'flux data table' // do not refactor or restrict to LayerTypes.RawFluxDataTable
```

Comment:
out of curiosity, why not? it's good to remember why we do things so we don't end up in a [monkeys on a ladder](https://www.youtube.com/watch?v=y-PvBo75PDo) situation

Reply:
This is for storybook and code examples.

Storybook examples are harder to work with when TypeScript restricts them to an enum. It causes storybook to import the enum, which by itself, is not a problem. But then this will also extend out to any other example code, including in the README.

The examples in the README become convoluted, since we are now referring to something that TypeScript requires rather than just a simple string. The README code is more approachable if we allow a string "flux data table" rather than require new users to already have a solid grasp of TypeScript.